lib/iex/lib/iex/helpers.ex

```diff
@@ -560,19 +575,20 @@ defmodule IEx.Helpers do
 
   defp expand_home(other), do: other
 
-  defp ls_print(_, []) do
+  defp tp(list, print_function \\ &String.pad_trailing/2)
```

Comment:
Since we keep this function private is doesn't need to have such cryptic name, let's name it `print_table` then. :bowtie: